src/commons/stores/user-store.js

```diff
@@ -0,0 +1,134 @@
+(function() {
+    "use strict";
+
+    angular
+        .module("TendrlModule")
+        .service("userStore", userStore);
+
+    /*@ngInject*/
+    function userStore($state, $q, userFactory) {
+        var store = this;
+        store.users = [];
+
+        store.getUserList = function() {
+            var list,
+                deferred;
+
+            deferred = $q.defer();
+            userFactory.getUserList()
+                .then(function(data) {
+                    if (data) {
+                        list = _setupUserListData(data);
+                        store.users = list;
+                    }
+                    deferred.resolve(list);
+                });
+
+            function _generateName(name) {
+                var userName = [],
```

Comment:
I think it should be an object, any specific reason of using list and then using it as an object?